### DISCUSSION TITLE
bitcoincore_rpc

### DISCUSSION THREAD

**Discussion by tcharding** [romanz/electrs#432] [General]
I see you have a few commits on `bitcoincore_rpc` repository already @romanz. Are there any technical reasons why `electrs` does not use `bitcoincore_rpc`?

Thanks,
Tobin.

**Comment by Kixunil** [comment]
It's [already used in `p2p` branch](https://github.com/romanz/electrs/blob/3bf5e5c266ecdbd10aa7709e495113521256e9f1/Cargo.toml#L25), which is WIP and will eventually become version 0.9.0. :)

We should not forget to close https://github.com/romanz/electrs/issues/57 once `p2p` lands...

**Reply by tcharding** [reply]
Oh nice, I'll take a look at that. Thanks.

**Comment by romanz** [comment]
It is released in 0.9.0 :)
